packages/graalvm-js/src/main/resources/META-INF/elide/embedded/runtime/js/runtime.current.json

```diff
@@ -6,9 +6,7 @@
   ],
   "artifacts": [],
   "image": {},
-  "scripts": [
-    { "path": "polyfills.js" }
-  ],
+  "scripts": [],
```

Comment:
I think we will still need polyfills for fetch... but you can also test with `elide ... --js:experimental-disable-polyfills`, which opts out of their use

Reply:
this looks like a bad rebase issue, I didn't manually remove these